lib/appoptics_apm/config.rb

```diff
@@ -65,17 +65,17 @@ def self.load_config_file
       config_file = File.join(Dir.pwd, 'appoptics_apm_config.rb')
       config_files << config_file if File.exist?(config_file)
 
-      return if config_files.empty?  # we use the defaults from the template in this case
-
-      if config_files.size > 1
-        AppOpticsAPM.logger.warn [
-          '[appoptics_apm/config] Multiple configuration files configured, using the first one listed: ',
-          config_files.join(', ')
-        ].join(' ')
+      unless config_files.empty? # we use the defaults from the template if there are no config files
+        if config_files.size > 1
+          AppOpticsAPM.logger.warn [
+                                     '[appoptics_apm/config] Multiple configuration files configured, using the first one listed: ',
+                                     config_files.join(', ')
+                                   ].join(' ')
+        end
+        load(config_files[0])
+        # sets AppOpticsAPM::Config[:debug_level], AppOpticsAPM.logger.level
+        set_log_level
```

Comment:
Should `set_log_level` be called even if the config file is empty as it can be set in an environment variable?

Reply:
yes, definitely, I'll correct it